src/logic/tokens/store/actions/activeTokensByBalance.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import type { Dispatch as ReduxDispatch } from 'redux'
+import { Set } from 'immutable'
+import { type GetState, type GlobalState } from '~/store'
+import updateActiveTokens from '~/routes/safe/store/actions/updateActiveTokens'
+import {
+  safeActiveTokensSelectorBySafe,
+  safeBlacklistedTokensSelectorBySafe,
+  safesMapSelector,
+} from '~/routes/safe/store/selectors'
+import fetchTokenBalanceList from '~/logic/tokens/api/fetchTokenBalanceList'
+import updateSafe from '~/routes/safe/store/actions/updateSafe'
+
+const activeTokensByBalance = (safeAddress: string) => async (dispatch: ReduxDispatch<GlobalState>, getState: GetState) => {
```

Comment:
since this is a function its name should include a verb, my suggestion is `activateTokensByBalance`

Reply:
indeed, thanks!

I changed it to the suggested name. Also fixed a couple of linting errors about _line too long_ and _no return value_ for the same action.

Reply:
Thank you!